added embed url generator application (#46)
tryretool/retool-app-exchange: embed-url-generator/README.md

# Embed URL Generator / Embedded App Tester

If you are using [Retool's Embed offering](https://docs.retool.com/docs/retool-embed), we've created a Retool application that lets you preview embedded apps by passing in different parameters, such as metadata, environments, and releases.

**Note**: We recommend using this application in *Incognito Mode* in your browser, to prevent conflicts between the sample embedded user and your original Retool login. We also recommend reading the docs linked above before using, especially the parts about passing in [metadata.](https://docs.retool.com/docs/embed-retool-apps#control-access-and-app-behavior-with-metadata)

<img src="./images/6_embed_url_app.png" >

## Setup
All of the setup steps are outlined within the application itself, as well as below. Once you set the below up, you can jump to the sixth step within the application.

### 0. Make sure you have the proper flags enabled

> This is only for customers who have our [Retool Embed](https://retool.com/products/embed) enabled. If you don't have it, but are interested in learning more, use [this link](https://retool.com/products/embed) to book some time with our team!

Reviewer: ```suggestion
This is only for customers who have [Retool Embed](https://retool.com/products/embed) enabled. If you don't have it, but are interested in learning more, use [this link](https://retool.com/products/embed) to book a demo with our team.
```
